internal/host/transition.go

```diff
@@ -419,7 +419,7 @@ func (th *transitionHandler) HasInstallationTimedOut(sw stateswitch.StateSwitch,
 	}
 	maxDuration, ok := InstallationProgressTimeout[sHost.host.Progress.CurrentStage]
 	if !ok {
-		maxDuration = InstallationProgressTimeout["DEFAULT"]
+		return false, errors.New("HasInstallationTimedOut invalid host progress state")
```

Comment:
What will happen if we return an error here?
I see this function is called from [here](https://sourcegraph.com/github.com/openshift/assisted-service@dd9c55e714084f745a94a23b5af0a266ab9d6fa7/-/blob/internal/host/statemachine.go#L243:33)

Reply:
The main question is how not to miss any steps. 
If we have a default then no one will update `InstallationProgressTimeout` so what will be the point of having it and not just have one default timeout? 
i think that if we have test infra running on each pr it will catch this case so we should return an error if we have a timeout on a step that not appear in the list right ?

Reply:
@eranco74 do we fail installation if we installer fail to report progress?

Reply:
@filanov no, the installation will not fail

Reply:
Then there is no point in failing this operation. it's good that you added all the steps, lets keep the default as well

Reply:
K